index.d.ts

```diff
@@ -23,7 +23,7 @@ interface $ {
 
 export const $: $
 
-export function cd(path: string)
+export function cd(path: string): Promise<void>
```

Comment:
This is not true. cd does not return any promise.

Reply:
I have updated `cd` to return `void`.